common/changes/@microsoft/rush/support-premajor-prepatch-version-bump-type_2024-11-07-06-42.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "Add support for premajor and prepatch bump types",
```

```suggestion
      "comment": "Add support for `premajor` and `prepatch` bump types in lockstep versions.",
```

This is only used by lockstepped versions, right?